packages/Molecules/Form/fields/fieldRplmarkup.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <div class="rpl-markup" v-html="schema.markup"></div>
+</template>
+
+<script>
+import { abstractField } from 'vue-form-generator'
+
+export default {
+  mixins: [abstractField]
+}
+</script>
+
+<style lang="scss">
+@import "~@dpc-sdp/ripple-global/scss/settings";
+@import "~@dpc-sdp/ripple-global/scss/tools";
+
+$tide-link-color: rpl-color('primary') !default;
+
+.rpl-markup {
+  @include rpl_typography;
+
+  // The below should be cleaned up. It's taken from TideWysiwyg component.
```

Comment:
As the WYSIWYG component exists in Nuxt, there is a duplication of styles. Ideally we would use the same component in both cases, but will require some effort and planning to move WYSIWYG back into Ripple.